src/cephadm/cephadmlib/daemons/smb.py

```diff
@@ -128,8 +153,20 @@ def envs(self) -> Dict[str, str]:
             'SAMBA_CONTAINER_ID': self.cfg.instance_id,
             'SAMBACC_CONFIG': json.dumps(self.cfg.config_uris()),
         }
+        # The CTDB support in sambacc project is considered experimental
+        # and it refuses to run without setting the following environment
+        # variable. This can be dropped once sambacc no longer needs it,
+        # possibly after the next sambacc release.
+        environ['SAMBACC_CTDB'] = 'ctdb-is-experimental'
+        environ['CTDB_NODES_LIST'] = '!' + ' '.join(_NODES_SUBCMD)
```

Comment:
Double check: `CTDB_NODES_LIST` requirement is no longer valid with the latest changes that have gone into Samba master, right? I hope everything works out of the new `nodes list` configuration parameter.

Reply:
Good catch. Will fix.